### PR TITLE
Update bookings-faq.yml

### PR DESCRIPTION
**Pull request by msbemba**
Added a question to FAQ about the retired payment option

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8736

File: microsoft-365/bookings/bookings-faq.yml [hunk 1]
```diff
@@ -68,7 +68,11 @@ sections:
           If a customer wants to disable access to Bookings for certain eligible licenses in their tenant, they can either use a group policy to restrict licenses or implement an OWA Mailbox policy that will restrict who is able to create new Bookings calendars.
           
           If you use an OWA Mailbox policy to disable access to Bookings, all users will have a Bookings license, however when they attempt to access the app, they'll only be successful if they're part of the policy or if they've been added to an existing Bookings calendar as staff. Details [here](turn-bookings-on-or-off.md) in the "Allow only selected users to create Bookings calendars" section.
-          
+      
+      - question: |
+           Can I link a payment option to my Bookings?
+        answer: |
+           No. The Microsoft owned payment system was retired.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-faq.yml]:
```suggestion
           No. The Microsoft-owned payment system is retired.
           
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**